infra/terraform/modules/data_access/github_webhooks_handler/tests/conftest.py

```diff
@@ -0,0 +1,140 @@
+from hashlib import sha1
+from moto import mock_sns
+import boto3
+import hmac
+import json
+import os
+
+import pytest
```

Comment:
Actually now that I commented I realised that `hashlib` is less than `hmac` so they're sorted by package name not by what you actually import.